### ISSUE TITLE
[Help]: select unwrap image

### ISSUE THREAD

**Issue by kjz1997** [gmtsar/gmtsar#635]
Hello ,teacher .
For better results, after program unwrap, I selected the interference pair. The original interference pair is 131, and the remaining 100 after selection. Then I deleted the unqualified interference pair in intf.tab. When running the sbas script, the program reported the following error：

compute RMS misfit

setting smoothing to   2.000 
setting radar wavelength to   0.055 m 
setting radar incidence angle to  36.000 degree 
setting range to 867286.000 m 
Memory Allocation Successful...
read table file ...
number of SAR scenes is 29 
 N and number of interferograms don't match! 

Then I rerun select_ The pairs script, although I know that the generation of interference pairs is only related to the time baseline and the spatial baseline, if the baseline parameters are changed, the combination of my interference pairs will also change, so what is the method to regenerate the filtered baseline map and the intf.in file.

**Comment by Xiaohua-Eric-Xu** [comment]
What is the sbas command you ran? The error seems to indicate you have input N number of interferograms but your intf.tab contains a different number of records.

**Comment by kjz1997** [comment]
@Xiaohua-Eric-Xu 
After unwrapping, I found that some interference pairs have poor quality. I deleted the interference pairs with poor quality in intf.tab, so the number of interference pairs in intf.tab is different from that in intf.in

**Comment by Xiaohua-Eric-Xu** [comment]
Then when you run sbas, you'll need to use the right number of interferograms that matches your intf.tab.

**Comment by kjz1997** [comment]
@Xiaohua-Eric-Xu 
But another error was reported
`sbas intf.tab scene.tab 100 29 2604 3470 -range 867286 -incidence 36 -wavelength 0.0554658 -atm 2.0 -smooth 2.0 -rms
`
Error：
compute RMS misfit

setting smoothing to   2.000 
setting radar wavelength to   0.055 m 
setting radar incidence angle to  36.000 degree 
setting range to 867286.000 m 
Memory Allocation Successful...
read table file ...
number of SAR scenes is 29 
Segmentation fault (core dumped)

**Comment by Xiaohua-Eric-Xu** [comment]
Seems you are using an earlier version of sbas.c that does not output needed memory. So here are some questions:

1) How much RAM does your machine have?
2) Is that enough to run your sbas command?

**Comment by kjz1997** [comment]
My computer has 16G RAM, and the sbas script only needs 10.3G to run. 
In addition, when I input the number of interference pairs correctly, It will report an error of N and number of interference  don ` t match. 
I want to know whether the number of interference pairs in the error refers to the number of interference pairs in the intf.tab file or the number of interference pairs in the intf.in file

**Comment by Xiaohua-Eric-Xu** [comment]
It should be the number of records in your intf.tab.

**Comment by Xiaohua-Eric-Xu** [comment]
> My computer has 16G RAM, and the sbas script only needs 10.3G to run.

I guess the question is, whether your computer allows sbas to take that much amount of memory, considering other stuffs are taking memory as well. To test this hypothesis, try downsample your gird with `gmt grdsample` and see whether the same error still persist.

**Comment by kjz1997** [comment]
But when I choose to use all images, although the running memory required by sbas is much higher than the memory of my computer, it will not end the process until half of the time, and the error reported is kill, not core dump.

So I suspect it is still the problem of scene.tab and intf.tab, but I have checked it and still can't find the problem.

This is my selected intf.tab and scene.tab
[intf.txt](https://github.com/gmtsar/gmtsar/files/10728163/intf.txt)
[scene.txt](https://github.com/gmtsar/gmtsar/files/10728164/scene.txt)

This is the original intf.tab and scene.tab
[intf.txt](https://github.com/gmtsar/gmtsar/files/10728201/intf.txt)
[scene.txt](https://github.com/gmtsar/gmtsar/files/10728206/scene.txt)

This is the memory condition of the computer and the memory required for operation

<img width="376" alt="捕获" src="https://user-images.githubusercontent.com/89446964/218626343-6affd58f-eb31-43e0-b3f5-8ba7574b1096.PNG">
Please have a look, teacher, thank you

**Comment by kjz1997** [comment]
The problem has been solved. Thank you for your time to answer my question.

**Comment by Xiaohua-Eric-Xu** [comment]
Could you report what caused the segmentation fault? And your solution was?

**Comment by kjz1997** [comment]
@Xiaohua-Eric-Xu 
My operation was wrong，when I selected the interferogram, all the interference pairs of some images were very poor, so I deleted all the interference pairs related to them from intf.tab, but did not delete them from scene.tab.

I have another problem. Due to the limited RAM of my computer, I have done sbas for IW1 and IW1 respectively. What script can be used to splice the final rate results
